app/controllers/client_api/users/user_teams_controller.rb

```diff
@@ -0,0 +1,85 @@
+module ClientApi
+  module Users
+    class UserTeamsController < ApplicationController
+      include NotificationsHelper
+      include InputSanitizeHelper
+      include ClientApi::Users::UserTeamsHelper
+
+      before_action :find_user_team, only: :leave_team
+
+      def leave_team
+        if user_cant_leave
```

Comment:
Predicate functions that return Boolean usually have '?' at the end of function name